### ISSUE TITLE
WCAG 4.1.2: Buttons must have discernible text (button[data-target="module-87"])

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/j4adminui#54]
#### Issue
`Buttons must have discernible text` ([button-name](https://dequeuniversity.com/rules/axe/3.3/button-name?application=msftAI))

It also doesn't appear to do anything
#### Target application
[Control Panel - kh - Administration](http://localhost/kh/administrator/index.php)
#### Element path
button[data-target="module-87"]
#### Snippet
`<button class="j-card-header-icon icon-chevron-down joomla-collapse-card-body" data-target="module-87"></button>`
#### How to fix
    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element's default semantics were not overridden with role="presentation"
      Element's default semantics were not overridden with role="none"
      Element has no title attribute or the title attribute is empty
#### Environment
Chrome version 78.0.3904.97
